lib/msf/core/payload/windows/meterpreter_loader.rb

```diff
@@ -28,6 +28,11 @@ def initialize(info = {})
       'PayloadCompat' => { 'Convention' => 'sockedi handleedi -https', },
       'Stage'         => { 'Payload'   => "" }
       ))
+    register_advanced_options(
+      [
+        OptBool.new('MeterpreterDebugBuild', [false, 'Use the Meterpreter debug build.'])
+      ]
+    )
```

Comment:
We'll want to add this into the base meterpreter options mixin, if we're landing this first I think we can grab what Simon has got over here https://github.com/rapid7/metasploit-framework/blob/5a5dfbfa4708e3448face4a295e846e16db8eb80/lib/msf/base/sessions/meterpreter_options.rb#L74-L77 - but it'll need the additional aliases list added for consistency with Java/python